<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" parallel="tests" name="Errorvalidation(failed)">
    <classes>
      <class name="selenium.PageObjModel.ErrorValidations">
        <methods>
          <include name="checkLoginError"/>
          <include name="launchApp"/>
          <include name="tearDown"/>
        </methods>
      </class> <!-- selenium.PageObjModel.ErrorValidations -->
    </classes>
  </test> <!-- Errorvalidation(failed) -->
  <test thread-count="5" parallel="tests" name="submited(failed)">
    <classes>
      <class name="selenium.PageObjModel.MainClassToBeCoverted">
        <methods>
          <include name="checkInOrders"/>
          <include name="launchApp"/>
          <include name="submitOrder" invocation-numbers="0 1"/>
          <include name="tearDown"/>
        </methods>
      </class> <!-- selenium.PageObjModel.MainClassToBeCoverted -->
    </classes>
  </test> <!-- submited(failed) -->
</suite> <!-- Failed suite [Suite] -->
